lib/components/map/connected-endpoints-overlay.js

```diff
@@ -17,9 +17,13 @@ const mapStateToProps = (state, ownProps) => {
   const activeSearch = getActiveSearch(state.otp)
   const query = activeSearch ? activeSearch.query : state.otp.currentQuery
   const showUserSettings = getShowUserSettings(state.otp)
-  const { from, to } = query
+  const { from, intermediatePlaces, to } = query
+  // FIXME: Splitting places by comma is broken in otp-ui:
+  // https://github.com/opentripplanner/otp-ui/issues/203
```

Comment:
This should be blocking as it is making it impossible to plan trips with 2+ intermediate places.

Reply:
It's breaking reloading an itinerary from a URL as well, but we don't have a good fix right now and it's not affecting our production code, so I don't think it should be blocking for this MVP at the moment.